src/js/index.js

```diff
@@ -1,12 +1,25 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
-var makeSubmitForm = require('./makeSubmitForm.jsx');
+var api = require('./api');
+var userSelect = require('./userSelect.jsx');
+var fiContainer = require('./fiContainer.jsx');
 
-var SubmitForm = makeSubmitForm('/submit', function(e){
-  console.log(e, 'file uploaded');
-});
 
 ReactDOM.render(
```

Comment:
Ahhh righto, thought that might be the case. Yeah we won't. We won't need to replace it though until we get the auth component hammered out, so I say leave it in for now.